File: Data Structures & Algorithms/remove-element/submission-1.py
class Solution:
    def removeElement(self, nums: List[int], val: int) -> int:
        
        # i = 0

        # for j in range(len(nums)):
        #      if nums[j] != val:
        #         nums[i] = nums[j]
        #         i += 1

        # return i

        i = 0
        for num in nums:
            if num != val: 
                nums[i] = num 
                i += 1
        return i

File: Data Structures & Algorithms/baseball-game/submission-1.cpp
class Solution {
public:
    int calPoints(vector<string>& operations) {
        vector<int> record;

        for (const string& curr : operations){
            if (curr == "+"){
                record.push_back(record.back()+record[record.size()-2]);
            } else if (curr == "D"){
                record.push_back(record.back()*2);
            } else if (curr == "C"){
                record.pop_back();
            } else {
                record.push_back(stoi(curr));
            }
        }

        int sum = 0;
        for (int num: record){
            sum += num;
        }

        return sum;
    }
};